package/MDAnalysis/auxiliary/base.py

```diff
@@ -545,7 +546,35 @@ def represent_ts_as(self, new):
     def __del__(self):
         self.close()
 
-    
+    def get_description(self):
+        """Get the args/kwargs necessary for dupicating the AuxReader
+
+        The resulting description may be passed to 'add_auxiliary' to 
+        reload an auxiliary into a trajectory.
+
+        Returns
+        -------
+        dict
+            Of args/kwargs that can be used to recreate the AuxReader
+        """
+        description = {key.lstrip('_'): val 
+                       for key, val in self.__dict__.items()
+                       if key in ['constant_dt', 'cutoff', '_time_col', 
```

Comment:
Ah yes - I did consider this but it got lost somewhere on the stuff-to-fix-up-later pile >_< . I'll change it now.